src/components/OnCartComponent.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from "react";
+import { FaShoppingCart } from "react-icons/fa";
+import CartContext from "../state/CartContext";
+
+//CSS
+import "../styles/OnCartComponent.css";
+
+const OnCartComponent = () => {
+    const {productsOnCart} = useContext(CartContext);
+
+  return (
+    <span className="On-cart">
+      <FaShoppingCart className="Shopping-cart" size={20} />
+      <span>{productsOnCart && productsOnCart.length > 0 ? productsOnCart.reduce((n, {quantity}) => n + quantity, 0): "0"}</span>
```

Comment:
I think we don't need the extra validation here
```suggestion
       <span>{productsOnCart.reduce((n, {quantity}) => n + quantity, 0)}</span>
```